src/load-test/prod/data.tf

```
@@ -0,0 +1,4 @@
+data "azurerm_virtual_network" "weu_common" {
+  name                = "${local.legacy_project}-vnet-common"
+  resource_group_name = "${local.legacy_project}-rg-common"
+}
```

```suggestion
}

data "azurerm_private_dns_zone" "privatelink_azurewebsites" {
  name                = "privatelink.azurewebsites.net"
  resource_group_name = "${local.legacy_project}-rg-common"
}
```

To support suggestion in src/load-test/prod/networking.tf